app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/Eav.php

```diff
@@ -787,12 +787,15 @@ public function setupAttributeMeta(ProductAttributeInterface $attribute, $groupC
     private function getAttributeDefaultValue(ProductAttributeInterface $attribute)
     {
         if ($attribute->getAttributeCode() === 'page_layout') {
-            $defaultValue = $this->scopeConfig->getValue(
-                'web/default_layouts/default_product_layout',
-                \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
-                $this->storeManager->getStore()
-            );
-            $attribute->setDefaultValue($defaultValue);
+            $defaultValue = $attribute->getDefaultValue();
+            if (!$defaultValue) {
```

Comment:
Please optimize this code. Nested ifs are looking weird. Why don"t you check it in one if statement?